config/settings.yml

```diff
@@ -176,7 +176,7 @@ form_10_10cg:
       client_id: ~
       client_secret: ~
       timeout: 30
-      async_timeout: 600
+      async_timeout: 120
```

Comment:
This value is overwritten in high envs, so it will need to be updated [in these three spots](https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fvsp-infra-application-manifests%20async_timeout&type=code) as well.